lib/reply.js

```diff
@@ -76,6 +84,11 @@ Reply.prototype.code = function (code) {
   return this
 }
 
+Reply.prototype.serializer = function (fn) {
+  this._serializer = fn
+  return this
+}
```

Comment:
Ahum, this custom serializer logic might require some more thinking.
We should let devs sets a serializer both at the `Reply`  level, but also at the route level.

For this change, I mean:

```js
reply.serializer('application-json', (res) => `"this is JSON"`)
```

👍  for the docs